specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerAppsJobs.json

```diff
@@ -0,0 +1,661 @@
+        "manualTriggerConfig": {
+          "type": "object",
+          "required": [
+            "completions",
+            "parallelism"
+          ],
+          "properties": {
+            "completions": {
+              "description": "Number of pods that needs to be successfully\ncompleted for a job to be complete.",
+              "format": "int32",
+              "type": "integer"
+            },
+            "parallelism": {
+              "description": "Number of parallel jobs that can run at a given\ntime.",
+              "format": "int32",
+              "type": "integer"
+            }
+          }
+        },
+        "scheduleTriggerConfig": {
+          "required": [
+            "completions",
+            "cronExpression",
+            "parallelism"
+          ],
+          "type": "object",
+          "properties": {
+            "completions": {
+              "description": "Number of pods that needs to be successfully\ncompleted for a job to be complete.",
+              "format": "int32",
+              "type": "integer"
+            },
+            "cronExpression": {
+              "description": "Cron formatted repeating schedule (\"* * * * *\")\nfor cronjobs.",
+              "type": "string"
+            },
+            "parallelism": {
+              "description": "Number of parallel jobs that can run at a given\ntime.",
+              "format": "int32",
+              "type": "integer"
+            }
+          }
+        },
+        "registries": {
+          "description": "Collection of private container registry credentials for containers used by the Container app",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RegistryCredentials"
+          },
+          "x-ms-identifiers": [
+            "server"
+          ]
+        },
+        "triggerType": {
+          "description": "Type of the job - Schedule | Event | Manual",
+          "enum": [
+            "Scheduled",
+            "Event",
+            "Manual"
+          ],
+          "type": "string",
+          "default": "Single",
+          "x-ms-enum": {
+            "name": "TriggerType",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "ContainerAppsJob": {
+      "type": "object",
+      "description": "Container App Job",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/TrackedResource"
+        }
+      ],
+      "properties": {
+        "extendedLocation": {
+          "$ref": "./CommonDefinitions.json#/definitions/ExtendedLocation"
+        },
+        "identity": {
+          "description": "managed identities for the Container App to interact with other Azure services without maintaining any secrets or credentials in code.",
+          "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentity"
+        },
+        "properties": {
```

Comment:
Aren't we missing `description` for this? See: https://github.com/Azure/azure-rest-api-specs/blob/release-app-Microsoft.App-2022-11-01-preview/specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/ContainerApps.json#L540